lib/src/main/java/com/basistheory/android/context/ExpressionsContext.kt

```diff
@@ -0,0 +1,20 @@
+package com.basistheory.android.context
+
+import java.util.UUID
+
+internal class ExpressionsContext {
+    companion object {
+        private var map = mutableMapOf<String, Any?>()
```

Comment:
are new instances of this being created for every dual write call?

otherwise we should clean this map and delete the ids already used after the call, to prevent running out of memory